test/stage2/test.zig

```diff
@@ -1569,4 +1565,27 @@ pub fn addCases(ctx: *TestContext) !void {
             \\}
         , "HeHelHellHello");
     }
+
+    {
+        var case = ctx.exe("double ampersand", linux_x64);
+
+        case.addError(
+            \\const a = if (true && false) 1 else 2;
+        , &[_][]const u8{":1:20: error: `&&` is invalid; note that `and` is boolean AND"});
+
+        case.addError(
+            \\pub fn main() void {
+            \\    const a = true;
+            \\    const b = false;
+            \\    const c = a & &b;
+            \\}
+        , &[_][]const u8{":4:17: error: incompatible types: 'bool' and '*const bool'"});
+
+        case.addCompareOutput(
+            \\pub fn main() void {
+            \\    const b: u8 = 1;
+            \\    const c = &&b;
```

Comment:
`c` is unused here which will cause a compile error with https://github.com/ziglang/zig/pull/9047 . Please fix it by adding _ = c; or changing it to _ = &&b;

Reply:
Sure, fixed